Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-github "ReactiveX/RxSwift" "3.0.1"
-github "daltoniam/Starscream" "2.0.2"
+github "ReactiveX/RxSwift" "3.4.0"
```

Comment:
I'm confused as to why you're using 3.4 here and 3.3.1 on your other PR. 
It should definitely be the same, whichever it is, or we're gonna conflict.